rotkehlchen/chain/evm/decoding/aave/common.py

```diff
@@ -329,6 +337,40 @@ def _decode_repay(
 
         return return_event, repay_event
 
+    def _decode_mint_events(self, context: DecoderContext) -> None:
+        """Decode possible mint events in transactions. Those are result of collateral/debt APY"""
+        mints = []
+        for tx_log in context.all_logs:
+            if tx_log.log_index >= context.tx_log.log_index:
+                break  # is possible to not have mint events. Also don't query after the pool event
+
+            if tx_log.topics[0] == MINT:
+                token = EvmToken(evm_address_to_identifier(
+                    address=tx_log.address,
+                    token_type=EvmTokenKind.ERC20,
+                    chain_id=self.evm_inquirer.chain_id,
+                ))
+                mints.append((
+                    hex_or_bytes_to_address(tx_log.topics[2]),  # onBehalfOf
+                    asset_normalized_value(  # amount minted
+                        amount=hex_or_bytes_to_int(tx_log.data[0:32]),
+                        asset=token,
+                    ),
+                    token,
+                ))
+
+        for event in context.decoded_events:
+            if (
+                (event.location_label, event.balance.amount, event.asset) in mints and
+                event.event_type == HistoryEventType.RECEIVE and
+                event.event_subtype == HistoryEventSubType.NONE and
+                event.counterparty is None
+            ):
+                event.event_subtype = HistoryEventSubType.RECEIVE_WRAPPED
+                resolved_asset = event.asset.resolve_to_asset_with_symbol()
+                event.notes = f'Receive {event.balance.amount} {resolved_asset.symbol} from {self.label}'  # noqa: E501
+                event.counterparty = self.counterparty
```

Comment:
Can there be multiple ones here? If not you need a break.

Don't you also need an else/nothing found case in which case you log an error?